app/workers/event_signup_reminder_worker.rb

```diff
@@ -0,0 +1,27 @@
+class EventSignupReminderWorker
+  include Sidekiq::Worker
+
+  def perform(event_signup_id, time_until)
+    event_signup = EventSignup.includes(:event_users).find(event_signup_id)
+    if notifyable(event_signup, time_until)
+      notify(event_signup)
+    end
+  end
+
+  private
+
+  def notifyable(event_signup, time_until)
+    return event_signup.present? &&
```

Comment:
You can skip the `return`, since ruby always returns the last line (in this case it's multiple lines, but it "counts" as one because of the `&&`). I think this method becomes a little bit clearer with the `return`, so maybe we should do an exception and keep it.